src/Context/ContextDefinition.php

```diff
@@ -70,10 +71,13 @@ public function toArray() {
    *
    * @return static
    *   The created definition.
+   *
+   * @throws \Drupal\rules\Exception\ClassDefintionException
+   *   If the required classes are not implemented.
    */
   public static function createFromArray($values) {
     if (isset($values['class']) && !in_array(ContextDefinitionInterface::class, class_implements($values['class']))) {
-      throw new \Exception('ContextDefinition class must implement ' . ContextDefinitionInterface::class . '.');
+      throw new ClassDefinitionException('ContextDefinition class must implement ' . ContextDefinitionInterface::class . '.');
```

Comment:
Since this is a direct extension of the core context system I think we should use \Drupal\Component\Plugin\Exception\ContextException here.